docs/sphinx/developers/matlab-dev.txt

```diff
@@ -187,9 +187,12 @@ To access physical voxel and stack sizes of the data:
       stackSizeX = omeMeta.getPixelsSizeX(0).getValue(); % image width, pixels
       stackSizeY = omeMeta.getPixelsSizeY(0).getValue(); % image height, pixels
       stackSizeZ = omeMeta.getPixelsSizeZ(0).getValue(); % number of Z slices
-      voxelSizeX = omeMeta.getPixelsPhysicalSizeX(0).getValue(); % in µm
-      voxelSizeY = omeMeta.getPixelsPhysicalSizeY(0).getValue(); % in µm
-      voxelSizeZ = omeMeta.getPixelsPhysicalSizeZ(0).getValue(); % in µm
+
+      voxelSizeX = omeMeta.getPixelsPhysicalSizeX(0).value(); % returns value in default unit
```

Comment:
Minor nitpick, but it might be good to use a different variable name here - otherwise the value would be overwritten in line 193.